maxi-blocks/maxi-blocks
# Issue-2691: Button Maxi break-line support

Fixes: #2691 
**How to test:** 
1- Add maxi-button block then add 02 text line, on the frontend should see the some as backend (02 lines), the some test for Text-maxi
